UpdatedImageProcessing/UpdatedImageProcessing/ShapeDetection/tester.py

```diff
@@ -4,7 +4,8 @@
 from PIL import Image
```

Comment:
What I heard from John, @DarkmatterVale , is that this PR is to get code necessary for @Inffzy . I believe he will remove this `tester.py` file at a later time.